packages/core/src/manifest.ts

```diff
@@ -41,6 +42,7 @@ export class Manifest {
 
   constructor(chainId: number) {
```

Comment:
The changes you made in this file can replaced by this change only:

```suggestion
  constructor(readonly chainId: number) {
```

(This is the way I'd do it.)